ccdproc/combiner.py

```diff
@@ -447,6 +447,73 @@ def average_combine(self, scale_func=ma.average, scale_to=None,
         # return the combined image
         return combined_image
 
+    def sum_combine(self, sum_func=ma.sum, scale_to=None,
+                        uncertainty_func=ma.std):
+        """
+        Sum combine together a set of arrays.
+
+        A `~ccdproc.CCDData` object is returned with the data property
+        set to the sum of the arrays. If the data was masked or any
+        data have been rejected, those pixels will not be included in the
+        sum. A mask will be returned, and if a pixel has been
+        rejected in all images, it will be masked. The uncertainty of
+        the combined image is set by the multiplication of summation of
+        standard deviation of the input by square root of number of images.
+
+        Parameters
+        ----------
+        scale_func : function, optional
+            Function to calculate the sum. Defaults to
+            `numpy.ma.sum`.
+
+        scale_to : float or None, optional
+            Scaling factor used in the sum combined image. If given,
+            it overrides `scaling`. Defaults to ``None``.
+
+        uncertainty_func : function, optional
+            Function to calculate uncertainty. Defaults to `numpy.ma.std`.
+
+        Returns
+        -------
+        combined_image: `~ccdproc.CCDData`
+            CCDData object based on the combined input of CCDData objects.
+        """
+        if scale_to is not None:
+            scalings = scale_to
+        elif self.scaling is not None:
+            scalings = self.scaling
+        else:
+            scalings = 1.0
+
+        # set up the data
+        data = sum_func(scalings * self.data_arr, axis=0)
+
+        # set up the mask
+        masked_values = self.data_arr.mask.sum(axis=0)
+        mask = (masked_values == len(self.data_arr))
+
+        # set up the deviation
+        uncertainty = uncertainty_func(self.data_arr, axis=0)
+        # Divide uncertainty by the number of pixel (#309)
+        # Commented out because it cancels the effect of 'Multiply uncertainty by square root of the number of images'
```

Comment:
I think there should be another line split. Even though 80 is just a "soft limit" I see no reason to make it much longer here.